homu/parse_issue_comment.py

```diff
@@ -16,7 +16,7 @@ def __init__(self, action):
     def approve(cls, approver, commit):
         command = cls('approve')
         command.commit = commit
-        command.actor = approver
+        command.actor = approver.lstrip('@')
```

Comment:
So would this also strip `@@RalfJung`? I guess GH names can't start with an `@` so it does not matter.

Reply:
Yeah apparently that's what `.lstrip(string)` does -- I agree it does not matter much.